spec/views/users/backup_code_setup/index.html.erb_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'users/backup_code_setup/index.html.erb' do
+  let(:user) { build(:user) }
+
+  before do
+    allow(view).to receive(:current_user).and_return(user)
+  end
+
+  it 'has a localized title' do
+    expect(view).to receive(:title).with( \
```

Comment:
Minor: Shouldn't need the `\` ?

```suggestion
    expect(view).to receive(:title).with(
```